packages/row/src/react/index.js

```diff
@@ -228,25 +228,29 @@ const formatActionBarWidth = ({ actionBar }) =>
 const Words = glamorous.div({ flex: 1 }, props => ({
   maxWidth: `calc(100% - ${formatImageWidth(props)} - ${formatActionBarWidth(
     props
-  )})`
+  )})`,
```

Comment:
these static styles can go on line 228, but `flex: 1`.  The later args to glamorous components are overrides.  I think this'll work.  Just convention-wise/readability-wise, I think we move it up.